NSS-Day-Cohort-61/team-1-GiffyGram
# Mrs register page

#### Changes Made
1. Modified file `Register.js` to include the RegisterForm function which creates a page for the user to add their information to the database. 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout mrs-register-page
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.

    >  When user clicks the sign up button, the user is taken to a page to input their information. On that sign up page is another button that return the user to the login page.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

close #7
